### PR TITLE
Additional bind-mounts (SOFTWARE-5513)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Hang on, bind-mounting /dev/fuse was for user-launched containers.

**Comment by matyasselmeci** [comment]
OK, ready.

File: docs/resource-sharing/os-backfill-containers.md [hunk 1]
```diff
@@ -46,11 +46,14 @@ In order to successfully start payload jobs:
...
    `-v /path/to/token:/etc/condor/tokens-orig.d/flock.opensciencegrid.org`.
    Replace `/path/to/token` with the full path to the token you obtained from the OSPool Token Registry.
-2. Set `GLIDEIN_Site` and `GLIDEIN_ResourceName` to match the site name and resource name that you registered in Topology,
+1. Set `GLIDEIN_Site` and `GLIDEIN_ResourceName` to match the site name and resource name that you registered in Topology,
    respectively.
-3. Set the `OSG_SQUID_LOCATION` environment variable to the HTTP address of your preferred Squid instance.
-4. _Strongly_recommended:_ Enable [CVMFS](#recommended-cvmfs) via one of the mechanisms described below.
-5. _Strongly recommended:_  If you want job I/O to be done in a separate directory outside of the container,
+1. Set the `OSG_SQUID_LOCATION` environment variable to the HTTP address of your preferred Squid instance.
+1. _If providing nVidia GPU resources:_ Bind-mount `/etc/OpenCL/vendors`, read-only.
```
Comment (brianhlin) [inline docs/resource-sharing/os-backfill-containers.md:52]:
```suggestion
1. _If providing NVIDIA GPU resources:_ Bind-mount `/etc/OpenCL/vendors`, read-only.
```

**Review by brianhlin — APPROVED** [review]
LGTM! One minor correction

**Pull request merged**